packages/otel/src/instrumentations/fetch.ts

```diff
@@ -330,65 +348,84 @@ export class FetchInstrumentation implements Instrumentation {
         if (this.shouldPropagate(url)) {
           const parentContext = context.active();
           const httpContext = traceApi.setSpan(parentContext, span);
-          propagation.inject(httpContext, options.headers || {}, HTTP_HEADERS_SETTER);
+          propagation.inject(
+            httpContext,
+            options.headers || {},
+            HTTP_HEADERS_SETTER,
+          );
         }
 
         if (attributesFromRequestHeaders) {
-            headersToAttributes(
+          headersToAttributes(
             span,
             attributesFromRequestHeaders,
             options.headers || {},
-            HTTP_HEADERS_GETTER
-            );
+            HTTP_HEADERS_GETTER,
+          );
         }
 
         try {
           const startTime = Date.now();
           const req = original.apply(this, [url, options, callback]);
 
-          req.prependListener('response', (res: IncomingMessage & { aborted?: boolean }) => {
-            const duration = Date.now() - startTime;
-            span.setAttribute("http.response_time", duration);
+          req.prependListener(
+            "response",
+            (res: IncomingMessage & { aborted?: boolean }) => {
+              const duration = Date.now() - startTime;
+              span.setAttribute("http.response_time", duration);
 
-            if (res.statusCode !== undefined) {
-              span.setAttribute(SemanticAttributes.HTTP_STATUS_CODE, res.statusCode);
-              if (res.statusCode >= 500) {
-                onError(span, `Status: ${res.statusCode}`);
+              if (res.statusCode !== undefined) {
+                span.setAttribute(
+                  SemanticAttributes.HTTP_STATUS_CODE,
```

Comment:
Let's be consistent and swap all `SemanticAttributes` with actual values or use your new constants.